matrix/tasks/reset.py

```diff
@@ -0,0 +1,12 @@
+import asyncio
+
+
+async def reset(context, rule, task, event=None):
```

Comment:
This touches on #24 and the "feature" of pre-deploying a model to make dev / test / iteration faster.